File: script/nodejs/println/fizz_buzz_println.js
// ==ClosureCompiler==
// @output_file_name fizz_buzz_println_compiled.js
// @compilation_level ADVANCED_OPTIMIZATIONS
// ==/ClosureCompiler==

var n = process.argv[2];
// const startTime = Date.now();
for(i=1; i<=n; i++){
    if(i%15 === 0){
        console.log("fizzbuzz");
    }else if(i%3 === 0){
        console.log("fizz");
    }else if(i%5 === 0){
        console.log("buzz");
    }else{
        console.log(i);
    }
}
// const endTime = Date.now();
// console.log("elapsed_time:"+(endTime - startTime)/1000+"[sec]");

File: script/c/println/fizz_buzz_println.c
#include <stdio.h>
#include <stdlib.h>
// #include <time.h>

int main(int argc, char *argv[])
{
   int n = atoi(argv[1]); // messy code

   int i;
   for (i = 1; i <= n; i++)
   {
      if (i % 15 == 0)
      {
         puts("fizzbuzz");
      }
      else if (i % 3 == 0)
      {
         puts("fizz");
      }
      else if (i % 5 == 0)
      {
         puts("buzz");
      }
      else
      {
         printf("%d\n", i);
      }
   }
   //  printf("elapsed_time:%.2f[sec]\n", ((double)clock()) / CLOCKS_PER_SEC);
}
